packages/swapper/src/manager/SwapperManager.ts

```diff
@@ -62,10 +73,50 @@ export class SwapperManager {
     return this
   }
 
-  async getBestSwapper(args: ByPairInput): Promise<Swapper<ChainId> | undefined> {
-    // TODO: This will eventually have logic to determine the best swapper.
-    // For now we return the first swapper we get from getSwappersByPair
-    return this.getSwappersByPair(args)[0]
+  async getBestSwapper(args: GetBestSwapperArgs): Promise<Swapper<ChainId> | undefined> {
```

Comment:
This is where the magic happens.

Reply:
popping this here for reference https://twitter.com/0xngmi/status/1610492831134629889?s=20&t=hBUmFYpEFEwNxQvLj1Fbag